packages/components/src/button-group/stories/index.story.tsx

```diff
@@ -10,7 +10,8 @@ import ButtonGroup from '..';
 import Button from '../../button';
 
```

Comment:
This is not very DRY but we're doing it for now so the deprecated status is more clear in the Storybook. We can probably remove when we implement something more systematic and visual, like a tag or something.

```suggestion
/**
 * ButtonGroup can be used to group any related buttons together.
 * To emphasize related buttons, a group should share a common container.
 *
 * This component is deprecated. Use `ToggleGroupControl` instead.
 */
```